packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1258,6 +1331,108 @@ describe(SucceedRelationshipAttributeAndNotifyPeerUseCase.name, () => {
     });
 });
 
+describe(ChangeDefaultRepositoryAttributeUseCase.name, () => {
+    test("should change default RepositoryAttribute", async () => {
+        const defaultAttribute = (
+            await services1.consumption.attributes.createRepositoryAttribute({
+                content: {
+                    value: {
+                        "@type": "GivenName",
+                        value: "My default name"
+                    }
+                }
+            })
+        ).value;
+
+        const desiredDefaultAttribute = (
+            await services1.consumption.attributes.createRepositoryAttribute({
+                content: {
+                    value: {
+                        "@type": "GivenName",
+                        value: "My new default name"
+                    }
+                }
+            })
+        ).value;
+
+        const result = await services1.consumption.attributes.changeDefaultRepositoryAttribute({ attributeId: desiredDefaultAttribute.id });
```

Comment:
Before the default RepositoryAttribute is changed, add expect statements that the `defaultAttribute` has currently `default: true` and the `desiredDefaultAttribute` has currently `default: undefined`.